### PR TITLE
Documentation added on github_issues.csv

### PR DESCRIPTION
**Pull request by propol**
Added the metrics' documentation on the github_issues.csv file in the schema folder. 
This is based to my use of the GrimoireLab tool during my thesis project :)
Also, consulted the other documented schema's to keep the same description style.

### CODE REVIEW

**Comment by propol** [comment]
Sure @alpgarcia , I will do it right now :)

File: schema/github_issues.csv [hunk 1]
```diff
@@ -1,64 +1,64 @@
...
+labels,keyword,true,"The labels assigned to an issue."
+metadata__enriched_on,date,true,"Date when the data were enriched."
+metadata__gelk_backend_name,keyword,true,"Name of the backend used to enrich the data."
+metadata__gelk_version,keyword,true,"Version of the backed used to enrich the data."
+metadata__timestamp,date,true,"Date when the item was stored in the ElasticSearch db as a raw index."
+metadata__updated_on,date,true,"Date when the item was updated on its original data source."
+ocean-unique-id,keyword,true,"Unique ID for all the datasets contained in the information pool."
+origin,keyword,true,"The original URL from which the repository was retrieved from."
+project_1,keyword,true,"Used if more than one project levels are allowed in the project hierarchy."
+project,keyword,true,"The project name."
```
Comment (alpgarcia) [inline schema/github_issues.csv]: I'd say just `Project name` as there usually are several projects.

File: schema/github_issues.csv [hunk 2]
```diff
@@ -1,64 +1,64 @@
...
+pull_request,boolean,true,"True/False if the item is a pull request or not."
+repository,keyword,true,"The repository's name."
```
Comment (alpgarcia) [inline schema/github_issues.csv]: As in the case of Project, I would say just `Repository name`

**Review by alpgarcia — CHANGES_REQUESTED** [review]
Please address my comments. Most of them are minor changes due to typos I found while reading the document.

Thanks a lot!

File: schema/github_issues.csv [hunk 3]
```diff
@@ -1,64 +1,64 @@
...
+assignee_data_org_name,keyword,true,"Assignee's organization name from SortingHat profile."
+assignee_data_user_name,keyword,true,"Assignee's username from SortingHat profile."
+assignee_data_uuid,keyword,true,"Assignee's UUID from SortingHat profile."
+assignee_domain,keyword,true,"Assignee's domain name from GitHub."
+assignee_geolocation,geo_point,true,"Assignee's global location using coordinates."
+assignee_location,keyword,true,"Assignee's geographical location."
+assignee_login,keyword,true,"Assignee's login name from GitHub."
+assignee_name,keyword,true,"Assignee's name from GitHub."
+assignee_org,keyword,true,"Assignee's organization name from GitHub."
+author_bot,boolean,true,"True/False if the autho is a bot or not."
```
[the remark this answers is not in the document]
Comment (propol) [reply schema/github_issues.csv]: Done!

File: schema/github_issues.csv [hunk 4]
```diff
@@ -1,64 +1,64 @@
...
+github_repo,keyword,true,"The name of the GitHub repository."
+grimoire_creation_date,date,true,"Issue creation date."
+id_in_repo,keyword,true,"The issue's ID in the repository it was created."
+id,long,true,"Issue's ID in GitHub."
+is_github_issue,long,true,"Used to seperate issues from other items such as pull requests."
+item_type,keyword,true,"The type of the item (issue/pull request)."
+labels,keyword,true,"The labels assigned to an issue."
+metadata__enriched_on,date,true,"Date when the data were enriched."
+metadata__gelk_backend_name,keyword,true,"Name of the backend used to enrich the data."
+metadata__gelk_version,keyword,true,"Version of the backed used to enrich the data."
```
[the remark this answers is not in the document]
Comment (propol) [reply schema/github_issues.csv]: Done

File: schema/github_issues.csv [hunk 5]
```diff
@@ -1,64 +1,64 @@
...
+metadata__timestamp,date,true,"Date when the item was stored in the ElasticSearch db as a raw index."
```
[the remark this answers is not in the document]
Comment (propol) [reply schema/github_issues.csv]: Done

File: schema/github_issues.csv (hunk 1 above)
Comment (propol) [reply schema/github_issues.csv]: Done

File: schema/github_issues.csv (hunk 2 above)
Comment (propol) [reply schema/github_issues.csv]: Done :)

**Comment by alpgarcia** [comment]
Thanks a lot for your contribution!

**Pull request merged**